packages/libs/error-stack/src/lib.rs

```diff
@@ -399,25 +407,26 @@
 //!  Feature   | Description                                                    | implies | default
```

Comment:
Do we want to add `experimental` as well? I'm unsure...

Reply:
Maybe, we should at least mention that `experimental` is not in semver guarantees. Maybe it would be better to call `unstable`? To bring it more in line with rust?

Reply:
I think both are fine, but `unstable` is probably slightly better yes.

Reply:
fixed in [3d098c4](https://github.com/hashintel/hash/pull/794/commits/3d098c4c91ac1fb1da6f9a10798787ba781e0738)